docs/deployment_patterns/how_to_run_a_checkpoint_in_airflow.md

```diff
@@ -14,59 +14,69 @@ This guide will help you run a Great Expectations checkpoint in Apache Airflow,
-    - The script can be run with `python great_expectations/uncommitted/my_checkpoint.py`
-    ```
+### Operator Parameters
 
-2. Navigate to the generated Python script and copy the content
-3. Create a method in your Airflow DAG file and call it from a PythonOperator:
+The operator has several optional parameters, but it always requires either a `data_context_root_dir` or a `data_context_config` and either a `checkpoint_name` or `checkpoint_config`.
 
-    ```python
-    def run_checkpoint():
-        # paste content from the checkpoint script here
+The `data_context_root_dir` should point to the `great_expectations` project directory generated when you created the project with the CLI. If using a `data_context_config`, a `DataContextConfig` must be defined, as in [this example](https://github.com/great-expectations/airflow-provider-great-expectations/blob/main/include/great_expectations/object_configs/example_data_context_config.py).
 
-    task_run_checkpoint = PythonOperator(
-        task_id='run_checkpoint',
-        python_callable=run_checkpoint,
-        dag=dag,
-    )
-    ```
+A `checkpoint_name` references a checkpoint in a series of subdirectories relative to the module (which is often the `great_expectations/checkpoints/` path), so that a `checkpoint_name = "taxi.pass.chk"` would reference the file `great_expectations/checkpoints/taxi/pass/chk.yml`. With a `checkpoint_name`, `checkpoint_kwargs` may be passed to the operator to specify additional, overwriting configurations. A `checkpoint_config` may be passed to the operator in place of a name, and can be defined like [this example](https://github.com/great-expectations/airflow-provider-great-expectations/blob/main/include/great_expectations/object_configs/example_checkpoint_config.py).
 
-Additional Resources
---------------------
+For a full list of parameters, see the `GreatExpectationsOperator` [documentation](https://registry.astronomer.io/providers/great-expectations/modules/greatexpectationsoperator).
 
-Please see [How to configure a New Checkpoint using "test_yaml_config"](../guides/validation/checkpoints/how_to_configure_a_new_checkpoint_using_test_yaml_config.md) for additional Checkpoint configuration and `DataContext.run_checkpoint()` examples.
+### Connections and Backends
 
+The `GreatExpectationsOperator` can run a checkpoint on a dataset stored in any backend compatible with Great Expectations. All that’s needed to get the Operator to point at an external dataset is to set up an [Airflow Connection](https://www.astronomer.io/guides/connections) to the datasource, and add the connection to your Great Expectations project, e.g. [using the CLI to add an Athena backend](https://docs.greatexpectations.io/docs/guides/connecting_to_your_data/database/athena). Then, if using a `DataContextConfig` or `CheckpointConfig`, ensure that the `"datasources"` field refers to your backend connection name.
```

Comment:
A couple of notes on this:
- I don't think an Airflow Connection is needed here (and I'm not sure how it could be used). I believe that it's just the Datasource that has to be specified.
- Instead of Athena (which is somewhat experimental), it would probably be better to link to our guide to Postgres instead (https://docs.greatexpectations.io/docs/guides/connecting_to_your_data/database/postgres). BigQuery, Snowflake, or Redshift would also be fine.
- I'm trying to think of how to make this as simple as possible. Something like
```suggestion
The `GreatExpectationsOperator` can run a checkpoint on data stored in any backend compatible with Great Expectations. All that’s needed to get the Operator to point at an external dataset is to configure and save a Great Expectations Datasource, which you can do with the CLI, (e.g. [with PostgreSQL](https://docs.greatexpectations.io/docs/guides/connecting_to_your_data/database/postgres)), or [in-memory](https://docs.greatexpectations.io/docs/guides/setup/configuring_data_contexts/how_to_instantiate_a_data_context_without_a_yml_file). Then, once you [create your ExpectationSuite](https://docs.greatexpectations.io/docs/guides/expectations/how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data) and [Checkpoint](https://docs.greatexpectations.io/docs/guides/validation/checkpoints/how_to_create_a_new_checkpoint), you'll be in business!
```

Reply:
You know, I haven't actually tried connecting to an external datasource without a connection configured... I just assumed Airflow would need it to make the connection. If that's the case, then might as well just remove the connections section all together, because that's GE-specific.